javascript/src/URDFLoader.js

```diff
@@ -410,6 +420,36 @@ class URDFLoader {
             target.urdfName = target.name;
             target.urdfNode = link;
 
+            // Extract the attributes
+            children.forEach(n => {
+
+                const type = n.nodeName.toLowerCase();
+                if (type === 'inertial') {
+                    const subNodes = [ ...n.children ];
+                    const origin = subNodes.find(sn => sn.nodeName.toLowerCase() === 'origin');
+                    const xyz = origin ? origin.getAttribute('xyz') : null;
+                    const rpy = origin ? origin.getAttribute('rpy') : null;
+                    const mass = subNodes.find(sn => sn.nodeName.toLowerCase() === 'mass');
+                    const inertia = subNodes.find(sn => sn.nodeName.toLowerCase() === 'inertia');
+                    target.inertial = origin || mass || inertia ? {
```

Comment:
Per the other comments, lets make sure this `inertial` field is already fully initialized with the correct defaults. And then this block can just overwrite any fields that are present in the file.